allure-cucumber-jvm/src/main/java/io/qameta/allure/cucumberjvm/AllureCucumberJvm.java

```diff
@@ -23,14 +23,9 @@
 import io.qameta.allure.util.ResultsUtils;
 import java.io.ByteArrayInputStream;
 import java.nio.charset.Charset;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```

Comment:
do not use wildcard imports

Reply:
cradle build throws error I don't use it
```[Coupling | ExcessiveImports] io.qameta.allure.cucumberjvm.(AllureCucumberJvm.java:1)
  A high number of imports can indicate a high degree of coupling within an object.
  https://pmd.github.io/pmd-5.5.4/pmd-java/rules/java/coupling.html#ExcessiveImports```

Reply:
Add `@SuppressWarnings("PMD. ExcessiveImports")` on class

Reply:
any advice here?

Reply:
> Add @SuppressWarnings("PMD.ExcessiveImports") on class